src/components/BudgetOverviewCard/BudgetOverviewCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { AccountBalance } from '@material-ui/icons';
+import { sumBy } from 'lodash';
+import * as React from 'react';
+import { FC } from 'react';
+import { FieldOverviewCard } from '../FieldOverviewCard';
+import { useCurrencyFormatter } from '../Formatters/useCurrencyFormatter';
+import { BudgetItemFragment } from './BudgetItem.generated';
+
+export interface BudgetItemCardProps {
+  budget?: BudgetItemFragment;
+  className?: string;
+}
+
+export const BudgetOverviewCard: FC<BudgetItemCardProps> = ({
+  className,
+  budget,
+}) => {
+  const currencyFormatter = useCurrencyFormatter();
+  const amount = sumBy(budget?.records, (record) => record.amount.value ?? 0);
+
+  return (
+    <FieldOverviewCard
+      className={className}
+      title="Project Budget"
+      viewLabel="Budget history"
+      data={
+        budget
+          ? {
+              updatedAt: budget?.createdAt,
+              value: currencyFormatter(amount),
+              to: `/${budget?.id}`,
```

Comment:
```suggestion
              to: `budget`,
```
The relative path should take us to `/project/{id}/budget`